### DISCUSSION TITLE
Question: Bull repeatable job concurrency issue

### DISCUSSION THREAD

**Discussion by aquiladev** [OptimalBits/bull#1992] [Q&A]
I have build repeatable job which repeats each minute. When I run it in a single instance of application, everything happen as expected. There is 1 worker per queue and 1 job per queue, so when processing is longer then 1 min, next repeat waits until current is completed.
But when I run multiple instances of applications, there is no waiting of completion of current one. Which is expected, because there are 1 worker per instance.
It is important for me that workers wait until completion of current execution. Is there a way to solve the issue?

I use bull v3

**Comment by manast** [comment]
I will answer with the same I did on gitter for reference:
I think the only way to solve this atm is by making sure you only have one worker running.
or not using repeatable jobs, just add the next job when you are done with previous job
if it executed faster than 1 minute just add a delay compensating, if it is slower than 1 minute add it without delay.
